requirements.txt

```diff
@@ -88,6 +88,7 @@ mock==1.3.0
 nameparser==0.5.7
 netCDF4==1.2.4
 nose==1.3.7
+nose-timer>=0.7.4,<1.0.0
```

Comment:
@cuahsimarko Just curious, is there a particular reason you put an upper version bound to nose-timer?

Reply:
I was taught to limit against major versions (breaking changes). I've never read this whole thing, but it's somewhere in here https://semver.org/